content/vault/v1.21.x (rc)/content/docs/sync/gcpsm.mdx

```diff
@@ -137,7 +139,23 @@ corresponds with the planned replication policy. The key rings and keys must be
 
 When using a global KMS key, it must be the only key set on the destination and the replication locations must remain unset, meaning
 it can only be used with using GCP's automatic replication. When specifying regional keys, a key must be set for each region in the
-replication location list. GCP key names are expected in the format of the entire resource ID, e.g. `projects/<project_id>/locations/<location_name>/keyRings/<key_ring_name>/cryptoKeys/<key_name>`. See the [API](#api) section for more details.
+replication location list.
+
+GCP key names are expected in the format of the entire resource name, e.g. `projects/<project_id>/locations/<location_name>/keyRings/<key_ring_name>/cryptoKeys/<key_name>`. Be aware that this is **case-sensitive**, with special attention on the `keyRings` and `cryptoKeys` portions.
+See the [API examples](/vault/api-docs/system/secrets-sync#sync-secrets-to-gcp-with-a-global-kms-key) for more details.
+
+Refer to the [GCP documentation](https://cloud.google.com/secret-manager/docs/cmek) for more details on setting this up. For brevity, the key parts are to create a service agent identity for
+the secretsmanager API and grant the service agent the correct IAM policy bindings with the
+`cloudkms.cryptoKeyEncrypterDecrypter` role. Be aware that this role must be granted to the service
+agent for each key and replication location being used.
```

Comment:
```suggestion
In general, the GCP key sections create a service agent identity for the
`secretsmanager` API and grant service agents the correct IAM policy bindings
with the `cloudkms.cryptoKeyEncrypterDecrypter` role. Be aware that you must
grant the cloud KMS role to the service agent for each key and replication
location you want to use.

Refer to the [GCP documentation](https://cloud.google.com/secret-manager/docs/cmek)
for more details on setting up your GCP key resources. 
```
Style correction: write in active voice, provide the summary before linking out to additional information